plugins/webp-uploads/picture-element.php

```diff
@@ -124,10 +122,10 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 			continue;
 		}
 		$picture_sources .= sprintf(
-			'<source type="%s" srcset="%s" sizes="%s">',
+			'<source type="%s"%s%s>',
 			esc_attr( $image_mime_type ),
-			esc_attr( $image_srcset ),
-			esc_attr( $sizes )
+			! empty( $image_srcset ) ? sprintf( ' srcset="%s"', esc_attr( $image_srcset ) ) : '',
```

Comment:
For PHPStan strict rules compliance:

```suggestion
			is_string( $image_srcset ) ? sprintf( ' srcset="%s"', esc_attr( $image_srcset ) ) : '',
```

Reply:
Updated in https://github.com/WordPress/performance/pull/1354/commits/6954f44727b86e1a25acee32432982b75c76469c